boards/kws/pico_spe/board.yml

```
@@ -0,0 +1,6 @@
+board:
+  name: pico_spe
+  full_name: SinglePairEthernet Pico
```

it looks like this should be the following, as this is the official/commercial name?

```suggestion
  full_name: Pico-SPE
```